<ui version="4.0" >
 <class>LastfmOptionsPage</class>
 <widget class="QWidget" name="LastfmOptionsPage" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>305</width>
    <height>317</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QGroupBox" name="rename_files" >
     <property name="title" >
      <string>Last.fm</string>
     </property>
     <layout class="QVBoxLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>2</number>
      </property>
      <item>
       <widget class="QCheckBox" name="use_track_tags" >
        <property name="text" >
         <string>Use track tags</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="use_artist_tags" >
        <property name="text" >
         <string>Use artist tags</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="rename_files_2" >
     <property name="title" >
      <string>Tags</string>
     </property>
     <layout class="QVBoxLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>2</number>
      </property>
      <item>
       <widget class="QLabel" name="ignore_tags_2" >
        <property name="text" >
         <string>Ignore tags:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="ignore_tags" />
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QLabel" name="ignore_tags_4" >
          <property name="sizePolicy" >
           <sizepolicy>
            <hsizetype>5</hsizetype>
            <vsizetype>5</vsizetype>
            <horstretch>4</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text" >
           <string>Join multiple tags with:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="join_tags" >
          <property name="sizePolicy" >
           <sizepolicy>
            <hsizetype>5</hsizetype>
            <vsizetype>0</vsizetype>
            <horstretch>1</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="editable" >
           <bool>true</bool>
          </property>
          <item>
           <property name="text" >
            <string/>
           </property>
          </item>
          <item>
           <property name="text" >
            <string> / </string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>, </string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QLabel" name="label_4" >
          <property name="sizePolicy" >
           <sizepolicy>
            <hsizetype>7</hsizetype>
            <vsizetype>5</vsizetype>
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text" >
           <string>Minimal tag usage:</string>
          </property>
          <property name="buddy" >
           <cstring>min_tag_usage</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="min_tag_usage" >
          <property name="suffix" >
           <string> %</string>
          </property>
          <property name="maximum" >
           <number>100</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>263</width>
       <height>21</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>use_track_tags</tabstop>
  <tabstop>ignore_tags</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
